docs/developer-guide/plugins.md

```diff
@@ -362,4 +362,32 @@ To make a single module provide multiple rules, export an array of plugin object
 
 ## Sharing plugins and plugin packs
 
-Use the `stylelint-plugin` keyword within your `package.json`.
+Use the `stylelint-plugin` keyword within your `package.json`. For example:
+
+```json
+{
+  "keywords": ["stylelint", "stylelint-plugin"]
+}
+```
+
+## Sharing configurations
```

Comment:
We will eventually need a dedicated `docs/developer-guide/config.md` page for _Writing configurations_.

Reply:
Probably yes. You can write it.